src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -79,7 +80,12 @@
                     "READ_PAIR_OPTICAL_DUPLICATES.\n" +
                     "\n" +
                     "Usage example: java -jar picard.jar MarkDuplicates I=input.bam \\\n" +
-                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n";
+                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n" +
+                    "\n" +
+                    "The program can take either coordinate-sorted or coordinate-sorted input, however the behaviour is slightly different." +
```

Comment:
As proof to how pedantic we've all become: you noticed that behavior was spelled wrong but didn't think about the actual meaning of the sentence:
"The program can take either coordinate-sorted or coordinate-sorted..."
I think there's a mistake in there...  :)